docs/util.md

```diff
@@ -18,13 +18,13 @@ Note, that all these classes employ TomP2P semantics. Think of these as a very t
 
 `FutureGet` extension classes: these work on all Get operations that TomP2P returns. Also includes all `BaseFuture` extensions.
 
-These fire on `BaseFuture`'s `onComplete`! Therefore, you have to check for success or failure yourself
+These fire on `BaseFuture`'s `onComplete`! Therefore, the success status still needs to be checked.
```

Comment:
They fire on `BaseFuture`'s `onComplete`. Therefore, the success status still needs to be checked.

Reply:
This is just the dot vs. the exclamation mark, right?